client/scss/components/_slider.scss

```diff
@@ -23,14 +23,16 @@ $slider-control-margin: 18px;
     margin-top: 0;
   }
 
-  &:before {
-    height: 60px;
-    display: block;
-    width: 100%;
-    content: '';
-    background: color('white');
-    margin-bottom: -5px;
-    clip-path: polygon(0% 75%, 0% 100%, 100% 100%, 100% 60%, 92% 85%, 70% 67%, 57% 90%, 45% 59%, 20% 72%);
+  &.slider__slider--with-wobbly-line {
+    &:before {
+      height: 60px;
+      display: block;
+      width: 100%;
+      content: '';
+      background: color('white');
+      margin-bottom: -5px;
+      clip-path: polygon(0% 75%, 0% 100%, 100% 100%, 100% 60%, 92% 85%, 70% 67%, 57% 90%, 45% 59%, 20% 72%);
+    }
```

Comment:
Could we generify this?
If you think the wobbly line can be pulled out - can we make a note of it somewhere to make sure we do it (on the conversation of reducing complexity).